test/language/generators/generator.expression.no-name.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015 Cubane Canada, Inc.  All rights reserved.
+// See LICENSE for details.
+
+/*---
+info: >
+    Generator can be declared with GeneratorExpression syntax
+es6id: 14.4
+author: Sam Mikes
+description: can create generator function expressions (no name)
+---*/
+
+var f = function *(a) { yield a+1; return; };
+
+assert.sameValue(f.name, '');
```

Comment:
Huh.  Okay, I will fix it.

To test for no name, I guess I need something like:

```
var a = [ function () {} ];
assert.sameValue(a[0].name, '')
```

Will that work?